src/migrations/20240830102144-onboarding-events.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+exports.up = function (db, cb) {
+    db.runSql(
+        `
+            CREATE TABLE onboarding_events (
+               event VARCHAR(50) NOT NULL,
```

Comment:
Can not be empty, 50 seems to be good enough. Do not need long definition

Reply:
events have 255 and it feels like a standard default. 50 feels weird